src/crl/interactivesessions/ShellSubprocess.py

```diff
@@ -87,7 +89,7 @@ def __init__(self, run_id, cmd, exception):
 
     def _get_result(self):
         if self._get_trace():
-            logging.log(7, self._get_trace())
+            LOGGER.debug(self._get_trace())
```

Comment:
This is changing loglevel from 7 to DEBUG (10)

Reply:
Possibly this is fine as this in an error scenario which would give useful information when checking debug logs of e.g. robot runs.